hadoop-hdds/erasurecode/src/main/java/org/apache/ozone/erasurecode/rawcoder/ByteBufferEncodingState.java

```diff
@@ -97,6 +101,7 @@ void checkBuffers(ByteBuffer[] buffers) {
       }
 
       if (buffer.remaining() != encodeLength) {
+        LOG.debug("buffer remaining is {} encodelength is {}", buffer.remaining(), encodeLength);
         throw new HadoopIllegalArgumentException(
             "Invalid buffer, not of length " + encodeLength);
```

Comment:
I wonder now if we need this debug message at all. Could we just adjust the exception to also include the `buffer.remaining()`, eg:

```
        throw new HadoopIllegalArgumentException(
            "Invalid buffer remaining " + buffer.remaining() 
            + ", not of length " + encodeLength);
```

The exception will propagate up the stack and give us the same information as the debug message anyway.